MainBrain/src/Controller/UnitekController/UnitekController.cpp

```diff
@@ -105,7 +105,7 @@ float UnitekController::calculateRpm(int speedValue)
  */
 uint16_t UnitekController::calculateSpeedSetPoint(float percent)
 {
-    float rpmSetPoint=(float)32767*percent;
+    float rpmSetPoint=(float)unitekModel->MAX_VALUE*percent;
```

Comment:
If you're planning on casting to a float and then casting back to an int, would it be simpler to cast to an int in the first place?